bridges/FM4Bridge.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+class FM4Bridge extends BridgeAbstract
+{
+	const MAINTAINER = 'joni1993';
+	const NAME = 'FM4 Bridge';
+	const URI = 'https://fm4.orf.at/tags/';
+	const CACHE_TIMEOUT = 1800; // 30min
+	const DESCRIPTION = 'Feed for FM4 articles by tags (authors)';
+	const PARAMETERS = array(
+		array(
+			'tag' => array(
+				'name' => 'Tag (author)',
+				'title' => 'Tag to retrieve (usually the author)',
+				'required' => true,
+			),
```

Comment:
Could you add "exampleValue" please? I don't know that exactly I need to input here to test it out.